Source/Core/AudioCommon/AlsaSoundStream.cpp

```diff
@@ -51,8 +57,11 @@ void AlsaSound::SoundLoop()
 	Common::SetCurrentThreadName("Audio thread - alsa");
 	while (m_thread_status.load() == ALSAThreadStatus::RUNNING)
 	{
+		std::unique_lock<std::mutex> lock(cv_m);
+		cv.wait(lock, [this]{return !m_muted || m_thread_status.load() != ALSAThreadStatus::RUNNING;});
+
 		m_mixer->Mix(reinterpret_cast<short *>(mix_buffer), frames_to_deliver);
-		int rc = m_muted ? 1337 : snd_pcm_writei(handle, mix_buffer, frames_to_deliver);
+		int rc = snd_pcm_writei(handle, mix_buffer, frames_to_deliver);
```

Comment:
I'm not sure if this is the way we should go. This will lead to an underflow in alsa on paused emulation. In my opinion, it's a) the best to stop the stream, but b) still fine to just output zero (ignore the m_muted flag).

By the way, I'm completely fine with dropping this flag and hoping for the audio backends to implement the Clear method in a good way.